CHANGELOG.md

```diff
@@ -9,6 +9,7 @@ Changelog for NeoFS Node
 
 ### Fixed
 - `neofs-cli object delete` command output (#3056)
+- Nil pointer dereferencing to N3 client connection (#3073)
```

Comment:
`panic during shutdown if N3 client connection is lost`? I think it's more correct here.